docs/reporters.md

```diff
@@ -124,7 +124,7 @@ This renders html code to view your tests in a browser. It includes all the test
 
 ## Automate Reporter
 
-The automate reporter type is a special reporter used with the Automate 2 suite. To use this reporter you must pass in the correct configuration via a json config `--json-config`.
+The automate reporter type is a special reporter used with the Automate 2 suite. To use this reporter you must pass in the correct configuration via a json config `--config`.
```

Comment:
```suggestion
The Automate reporter type is a special reporter used with Chef Automate (Version 2). To use this reporter you must pass in the correct configuration via a json config `--config`.
```

Reply:
```suggestion
The automate reporter type is a special reporter used with the [Chef Automate](https://automate.chef.io/). To use this reporter you must pass in the correct configuration via a json config `--config`.
```

Reply:
For the first mention I've placed `automate` in backticks, because it refers to the technical name of the reporter, and case matters.  I've placed the second reference in a link.